build.gradle.kts

```diff
@@ -176,27 +178,55 @@ kotlin {
     }
 }
 
-val generatedDir by extra("$projectDir/generated")
-
-protobuf {
-    generatedFilesBaseDir = generatedDir
-    generateProtoTasks {
-        all().forEach { task ->
-            task.builtins.maybeCreate("kotlin")
-            task.plugins {
-                remove("grpc")
-            }
-        }
+/**
+ * Suppress the "legacy" validation from McJava in favour of tha based on ProtoData.
+ */
+modelCompiler {
+    // The below arrangement is "unusual" `java { }` because it conflicts with
+    // `java` of type `JavaPluginExtension` in the `Project`.
```

Comment:
@armiol, @dmdashenkov, this is so because we're in a single-module project. This issue should be taken into the account for elaborating new DSL for Spine Model compiler.

We probably can take the standard `java` extension and extend it with our properties for code generation.